packages/scene-composer/src/common/entityModelConstants.ts

```diff
@@ -47,3 +47,6 @@ export const layerPropertyNames = {
   queryString: 'queryString',
   relationshipName: 'relationshipName',
 };
+
+// Dynamic Scenes
+export const DYNAMIC_SCENE_CAPABILITY = 'DYNAMIC_SCENE';
```

Comment:
let's make it like this since we support both capabilities currently
```
export enum SceneCapabilities {
  MATTERPORT = 'MATTERPORT',
  DYNAMIC_SCENE = 'DYNAMIC_SCENE',
}
```

Reply:
Added and pushed!